app/models/beneficiaries/RoleInCompany.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.beneficiaries
+
+import models.{Enumerable, WithName}
+import viewmodels.RadioOption
+
+sealed trait RoleInCompany
+
+object RoleInCompany extends Enumerable.Implicits {
+
+  case object Director extends WithName("Director") with RoleInCompany
+  case object Employee extends WithName("Employee") with RoleInCompany
+  case object NA extends WithName("NA") with RoleInCompany
```

Comment:
again should these be camel case? Just thinking about how that was agreed for journey testing when using the radio button IDs

Reply:
These are (or should be) internal values matching the JSON. We need a different presentation for "NA" at least, but that is pending from Rachel. I might just drop in what she has said so far, about what she remembers but hasn't confirmed.

Reply:
Yep I realised this after all the pipelines failed! I merged in the values matching the schema, and the "different presentation for NA" if I understand what you mean correctly is dealt with in AnswerRowConverter line 52